File: Test1/test1/src/app/model/employee.ts
import { Dependant } from "./dependant";

type undefinedString = string | undefined;

export class Employee {
    firstName : string | undefined;
    lastName : string | undefined;
    dependants : Dependant[] | undefined;
    cost : number =0
    constructor(public fname: string, public lname:string){
        this.firstName=fname;
        this.lastName = lname;
        this.dependants=[];
    }
}

File: BenefitsAPIs/BenefitsAPIs/Model/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BenefitsAPIs.Model
{
    public class Employee
    {
        public string FirstName {get;set;}
        public string LastName { get; set; }
        public List<Dependant> dependants { get; set; }
        public int Cost { get; set; }
        public Employee()
        {
            dependants = null;
        }
    }
}
